modules/nw-metallb-configure-return-traffic-proc.adoc

```diff
@@ -157,7 +160,7 @@ spec:
     - first-pool
   peers:
     - frrviavrf <1>
-  nodeSelectors:
+  nodeSelector:
```

Comment:
this one is fine, the problem is `BGPAdvertisement` with `nodeSelector` and not `nodeSelectors`, such in `nw-egress-service-ovn.adoc`

Reply:
@oribon guess I am bit confused as it states nodeSelectors here: https://docs.openshift.com/container-platform/4.17/networking/metallb/metallb-configure-bgp-peers.html#nw-metallb-bgppeer-cr_configure-metallb-bgp-peers